neqo-transport/src/qlog.rs

```diff
@@ -22,57 +25,46 @@ use crate::path::PathRef;
 use crate::stream_id::StreamType as NeqoStreamType;
 use crate::tparams::{self, TransportParametersHandler};
 use crate::tracking::SentPacket;
-use crate::Version;
 
 pub fn connection_tparams_set(qlog: &mut NeqoQlog, tph: &TransportParametersHandler) {
     qlog.add_event(|| {
         let remote = tph.remote();
-        Some(Event::transport_parameters_set(
-            None,
-            None,
-            None,
-            None,
-            None,
-            None,
-            remote
+        let ev_data = EventData::TransportParametersSet(
+            qlog::events::quic::TransportParametersSet {
+                owner: None,
+                resumption_allowed: None,
+                early_data_enabled: None,
+                tls_cipher: None,
+                aead_tag_length: None,
+                original_destination_connection_id: remote
```

Comment:
Do you implement Default for TransportParametersSet?  Because then you could use `..Default::default()` rather than providing all these `None`-valued arguments.